database/factories/LessonUserFactory.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Database\Factories;
+
+use App\Models\LessonUser;
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+class LessonUserFactory extends Factory
```

Comment:
chỗ này cũng tương tự như trên, phần factory kiểu quan hệ này không nên định nghĩa https://laravel.com/docs/8.x/database-testing#belongs-to-relationships